README.md

```diff
@@ -75,9 +75,13 @@ docker run --env-file ./.env my-test-image tests/endtoend
 ### Running the application on Azure
 The URL for the app on Azure is : http://madhuazuretodo.azurewebsites.net/
 
+### Running the application on Azure via Terraform
+The URL for the app on Azure via Terraform is : https://madhuterraformhelloapp.azurewebsites.net
```

Comment:
Nice, it would be good to also mention how a new developer could work with terraform if they needed to